packages/flutter/lib/src/widgets/debug.dart

```diff
@@ -132,6 +132,15 @@ bool debugProfileBuildsEnabled = false;
 ///    for every widget and has a higher overhead cost.
 bool debugProfileBuildsEnabledUserWidgets = false;
 
+/// Adds debugging information to [Timeline] events related to [Widget] builds.
+///
+/// This flag will only add [Timeline] event arguments for debug builds.
+/// Additional arguments will be added for the "BUILD" [Timeline] event and for
```

Comment:
Let's also document what these arguments are, e.g. additional stats around widget dirtyness and for widgets you get the diagnostics information provided by the widget (i.e. the properties of the widget). The latter can make it easier to identify a certain widget in the timeline.